README.markdown

```diff
@@ -4,22 +4,22 @@ Enables instructors to create questions with free-text responses.
 ## TODO List:
```

Comment:
Yeah, the checklist is just meant to be a "mini guide" during development; we want to update it prior to merge.